tree/dataframe/inc/ROOT/RDFActionHelpers.hxx

```diff
@@ -100,6 +101,10 @@ public:
    void Initialize() { /* noop */}
 
    void Finalize() { /* noop */}
+
+   std::string GetActionName(){
```

Comment:
perhaps here we may return a string_view to avoid fresh creation of strings?

Reply:
I think this represents a problem, as many of these strings are later manipulated for printing. Maybe it's better to use strings and pass them around as const ref whenever possible